GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -225,4 +230,12 @@ extension ChatWrapperViewController: ChatControllerDelegate {
     func didClickLink(_ url: String) {
         print("Link \(url) was pressed in the chat")
     }
+    
+    func didLogoutChat() {
+        DispatchQueue.main.async { [weak self] in
```

Comment:
is this the best location to switch to main thread?
dismissChat calls chatController.terminate()
Is it a requirement for chatController.terminate() to be called from main thread?